File: uwu.php
<?php
$uwu = "owo";

function uwuify($uwu) {
    echo "UwU what is this? " . $uwu . "\n";
}

uwuify($uwu);

for ($i = 1; $i <= 5; $i++) {
    echo "UwU count: " . $i . "\n";
}

$uwuArray = ["uwu", "owo", "uwu", "owo"];

foreach ($uwuArray as $index => $uwu) {
    echo "UwU index " . ($index + 1) . ": " . $uwu . "\n";
}

function uwuMath($a, $b, $length) {
    return ($a + $b) * $length;
}

echo "UwU math result: " . uwuMath(3, 4, count($uwuArray)) . "\n";

if ($uwu == "owo") {
    echo "UwU condition met!\n";
} else {
    echo "Not UwU enough!\n";
}
?>

File: uwu.java
public class UwU {
    public static void main(String[] args) {
        String uwu = "owo";

        System.out.println("UwU what is this? " + uwu);

        for (int i = 1; i <= 5; i++) {
            System.out.println("UwU count: " + i);
        }

        String[] uwuArray = {"uwu", "owo", "uwu", "owo"};

        for (int i = 0; i < uwuArray.length; i++) {
            System.out.println("UwU index " + (i + 1) + ": " + uwuArray[i]);
        }

        int result = uwuMath(3, 4, uwuArray.length);
        System.out.println("UwU math result: " + result);

        if ("owo".equals(uwu)) {
            System.out.println("UwU condition met!");
        } else {
            System.out.println("Not UwU enough!");
        }
    }

    public static int uwuMath(int a, int b, int length) {
        return (a + b) * length;
    }
}
